### PR TITLE
chore: Add 2022 post

### PR DESCRIPTION
**Pull request by gaocegege**
Signed-off-by: Ce Gao <cegao@tensorchord.ai>

### CODE REVIEW

**Comment by gaocegege** [comment]
Close #236

File: docs/blog/2022.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+Check out our official [documentation](https://envd.tensorchord.ai/) for more information or join us on [Discord](https://discord.gg/KqswhpVgdU) and let's talk about it together. See you there!
+
+## Future plans
+
+As we enter 2023, there is still plenty of work to be done in order for us to fully optimize the system. We are dedicating our focus on building performance, as well as fine-tuning and serving large models - all essential tasks that will help drive further progress this year!
+
+Not only that, get ready to experience the power of envd with our brand new online demo server running on Kubernetes! If you're already using it, tell us about the experience in [Discord](https://discord.gg/KqswhpVgdU).
+
+It's no secret that the machine learning process still needs a few tweaks here and there.  But don't worry - we won't stop until it runs like an absolute dream!  With any luck, 2023 holds something special in store for us all – so stay tuned to see what exciting new developments we can bring your way.
```
Comment (VoVAllen) [inline docs/blog/2022.md:50]: Author Footer missed

File: docs/blog/2022.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+introduction: 'Do cool things that matter | Co-founder @tensorchord | Co-chair @kubeflow | ex-@Tencent, @bytedance, @caicloud'
+---
+
+# TensorChord: 2022 in Review
+
+In just six months, TensorChord has accomplished amazing feats with the launch of the first open source project [envd](https://github.com/tensorchord/envd)! To celebrate this success and look ahead to 2023, we wanted to share our journey so far and provide insight into what's in store for us over the next year!
+
+## Open Source community
+
+On June 18th, 2022 envd officially opened its doors to the public. We were overwhelmed with feedback from our community and we could not be more delighted about it! Since then, we've had 100+ Pull Requests in just over 4 months - a remarkable achievement by any standard. Even better news: With 1024 GitHub stars in under 5 months since open source release.
```
Comment (kemingy) [inline docs/blog/2022.md:15]: Shall we use `envd` or plain "envd"?

File: docs/blog/2022.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+"numpy",
+])
+shell("zsh")
+```
+
+Check out our official [documentation](https://envd.tensorchord.ai/) for more information or join us on [Discord](https://discord.gg/KqswhpVgdU) and let's talk about it together. See you there!
+
+## Future plans
+
+As we enter 2023, there is still plenty of work to be done in order for us to fully optimize the system. We are dedicating our focus on building performance, as well as fine-tuning and serving large models - all essential tasks that will help drive further progress this year!
```
Comment (kemingy) [inline docs/blog/2022.md:46]:
```suggestion
As we enter 2023, there is still plenty of work for us to be done in order to fully optimize the system. We are dedicating our focus on building performance, as well as fine-tuning and serving large models - all essential tasks that will help drive further progress this year!
```

**Review by kemingy — COMMENTED** [review]
It's hard to review the Chinese blog.

File: docs/blog/2022.md (hunk 3 above)
Comment (VoVAllen) [inline docs/blog/2022.md:46]: dedicating our focus on building performance -> dedicating our focus to build speed

File: docs/blog/2022.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+Not only that, get ready to experience the power of envd with our brand new online demo server running on Kubernetes! If you're already using it, tell us about the experience in [Discord](https://discord.gg/KqswhpVgdU).
```
Comment (VoVAllen) [inline docs/blog/2022.md:48]:
```suggestion
Not only that, but get ready to experience the power of envd with our brand-new online demo server running on Kubernetes! If you're already using it, tell us about the experience at [Discord](https://discord.gg/KqswhpVgdU).
```

File: docs/blog/2022.md (hunk 1 above)
Comment (gaocegege) [reply docs/blog/2022.md:50]: It's in the head line.

**Comment by gaocegege** [comment]
Comments are addressed, please take a look.
Thanks for your time :smile:

**Comment by gaocegege** [comment]
> It's hard to review the Chinese blog.

Why is it hard?

File: docs-zh/blog/2022.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+introduction: 'Ce Gao 是 envd 的维护者之一。与此之外，他还是机器学习基础设施开源项目 Kubeflow 的 Co-chair。他主要关注机器学习的模型训练、自动机器学习等领域。'
+---
+
+# TensorChord: 2022 in Review
+
+尽管 TensorChord 还很年轻，但自去年六月底推出了我们的第一个开源项目 [envd](https://github.com/tensorchord/envd) 以来，短短的六个月内取得了令人惊喜的结果。为了庆祝我们迄今为止取得的进展，并展望 2023 年的前景，我们想分享今年的工作，并简单讨论我们看到的下一步方向。
+
+## 开源社区
+
+2022 年 6 月 18 日，我们开源了 envd 并发布了第一个正式版本 0.1.0。到目前为止，我们收获了来自社区的大量反馈，我们也在不断地改进我们的产品，保持着以一周为单位的迭代周期与版本发布节奏。在开源的一周后，我们合并了第 100 个 PR；在 10 月 21 日，我们获得了第 1024 个 GitHub star。我们的社区正在不断壮大，我们也在不断地学习如何与社区进行有效的沟通。
```
Comment (kemingy) [inline docs-zh/blog/2022.md:15]: `envd`

**Review by kemingy — COMMENTED** [review]
The Chinese version is hard to review because I don't know the standard.

If we choose the Chinese exam level, then there will be tons of mistakes to fix.

File: docs-zh/blog/2022.md (hunk 5 above)
Comment (gaocegege) [reply docs-zh/blog/2022.md:15]: Actually, the Chinese content is also generated by jasper

**Comment by gaocegege** [comment]
Typos in Chinese are also resolved.

**Comment by gaocegege** [comment]
Could we merge this PR?

**Pull request merged**